packages/flutter_tts/tizen/src/flutter_tts_tizen_plugin.cc

```diff
@@ -241,14 +240,17 @@ class FlutterTtsTizenPlugin : public flutter::Plugin {
     std::unique_ptr<flutter::EncodableValue> args =
         std::make_unique<flutter::EncodableValue>(true);
     if (current == TTS_STATE_PLAYING) {
+      plugin->speaking_ = true;
       if (previous == TTS_STATE_READY) {
         plugin->channel_->InvokeMethod("speak.onStart", std::move(args));
       } else if (previous == TTS_STATE_PAUSED) {
         plugin->channel_->InvokeMethod("speak.onContinue", std::move(args));
       }
     } else if (current == TTS_STATE_PAUSED) {
+      plugin->speaking_ = false;
```

Comment:
Maybe we need something like this here and in the next:

```cpp
if (previous == TTS_STATE_PLAYING && plugin->await_speak_completion_) {
  plugin->result_->Success();
}
```

Reply:
Thank you for suggestion, please review again